src/lib/punctuation/dash.js

```diff
@@ -143,27 +162,37 @@ export function fixHyphenBetweenWordAndPunctuation(string, locale) {
 export function fixDashBetweenCardinalNumbers(string, locale) {
 	/* [1] Match the pattern twice, replace it with enDash adepts */
 	let pattern =
-				"(" + locale.cardinalNumber + ")"
-			+ "([" + locale.spaces + "]?"
-			 + "[" + locale.hyphen + locale.enDash + locale.emDash + "]"
-			 + "[" + locale.spaces + "]?)"
-			+ "(" + locale.cardinalNumber + ")";
+		'(' +
+		locale.cardinalNumber +
+		')' +
+		'([' +
+		locale.spaces +
+		']?' +
+		'[' +
+		locale.hyphen +
+		locale.enDash +
+		locale.emDash +
+		']' +
+		'[' +
+		locale.spaces +
+		']?)' +
+		'(' +
+		locale.cardinalNumber +
+		')'
```

Comment:
Is it possible to set linter so that it does not fix formatting of the patterns? I prefer to read patterns in chunks that are usually groups of patterns.

Reply:
Same reason as stated above. I guess the only way to go around this is annotating this line with `// prettier-ignore`